Server/src/config/envs/swarm.docker.ts

```diff
@@ -48,10 +48,15 @@ export function createDockerConfig() {
         host: '0.0.0.0',
         port: 8088,
       },
+      tenant: {
+        endpointPrefix: 'tenant',
+        host: '0.0.0.0',
+        port: 8089,
```

Comment:
i updated the port because i wanted to keep the alphabetical ordering of the modules -- if it's preferred to not keep the alphabetical ordering and instead add the tenant to the "end" of the list, thereby making it port 8090, i can do so, let me know!

Reply:
I think keeping the alphabetical ordering is good. But changing the ports seems risky, as others and integration tests could already depend on the ports of certain modules.

Reply:
makes sense! then i've kept the alphabetical ordering, but the ports will still be in order of when the modules were created (meaning tenant now inherits 8090 and transaction returns to 8089) 👍